### DISCUSSION TITLE
No Data for Stock

### DISCUSSION THREAD

**Discussion by Akede-a** [ghostfolio/ghostfolio#1949] [Q&A]
Hi, I selfHosted Ghostfolio, wich works fine. 
Problem is it seems Like I cant find Stocks in the Activity Tab while adding Transactions. 
It Just does not find any Stocks, tried searching with ISIN and with Name... No results

Crypto is fine, it find's the Names and historical datA.

Am I Missing a Setting or do I have some Problem with the Database? 
Some ideas in how to fix this would be nice.

![Screenshot_2023-05-08-14-00-39-022_com android chrome-edit](https://user-images.githubusercontent.com/74055814/236818518-71919061-ca99-46dd-b36f-f8d808232d39.jpg)
Not Apple ISIN but dosn't matter wich ISIN or Name I search.
![Screenshot_2023-05-08-13-58-38-200_com android chrome-edit](https://user-images.githubusercontent.com/74055814/236818278-5bfd10fd-9e6b-48ff-a7b3-3ec5b24844c5.jpg)

**Comment by dtslvr** [comment]
Hi @Akede-a

There is an open ticket for this issue [here](https://github.com/ghostfolio/ghostfolio/issues/1948).

**Reply by Akede-a** [reply]
I don't get any Error, Like the other Ticket, so I asumed it would be a different Bug, sorry.
